code-challenges.test.js

```diff
@@ -58,3 +133,17 @@ var numbersToAdd3 = []
 
 
 // b) Create the function that makes the test pass.
+
+// Declare a function called addSumNums that takes in an array as a parameter
+// return the array using .map for iteration
+// .map should start at index 0 then take the sum and assign it to the next value, then take the sum and add it with that value
+
+const addSumNums = (array) => {
+  return array.map((sum => value => sum += value)(0))
+}
+console.log(addSumNums(numbersToAdd2));
+
+// #3 made me want to cry
+// I had nested for loops and if statements at first and I was like EwMG so chaotic and messy! SOooOo I did a google research deep dive and found .map can use math properties! Thanks Stackoverflow!
```

Comment:
This is a clean solution but I like seeing your creativity more! I don't mind having this brought in as well.